Formula/neomutt.rb

```diff
@@ -0,0 +1,37 @@
+class Neomutt < Formula
+  desc "Email reader with support for Notmuch (fast indexing), News (NNTP) and much more"
```

Comment:
@flatcap It's marked as error in brew audit because it's >80 chars.
Does this sound still catchy enough to you?

Reply:
desc "Email reader with support for Notmuch, NNTP and much more"